tests/test_config.py

```diff
@@ -32,6 +32,7 @@ def _assert_defaults(self, cfg: config.Config):
         cache_dir = os.path.join(xdg_cache_home(), package_name)
         data_dir = os.path.join(xdg_data_home(), package_name)
         default_model = f"{cache_dir}/models/merlinite-7b-lab-Q4_K_M.gguf"
+        default_sdg_model = f"{cache_dir}/models/mistral-7b-instruct-v0.2.Q4_K_M.gguf"
```

Comment:
can you also rename `default_model` to `default_chat_model` to differentiate the purpose?

Reply:
sure